samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -266,9 +271,11 @@ AppController.prototype.transitionToWaiting_ = function() {
 
   this.transitionToWaitingTimer_ = setTimeout(function() {
```

Comment:
Makes sense. Will do that.

On Fri, Jan 23, 2015 at 1:26 PM, Justin Uberti <notifications@github.com>
wrote:

> In samples/web/content/apprtc/js/appcontroller.js
> <https://github.com/GoogleChrome/webrtc/pull/420#discussion_r23481519>:
>
> > @@ -266,9 +271,11 @@ AppController.prototype.transitionToWaiting_ = function() {
> >
> >    this.transitionToWaitingTimer_ = setTimeout(function() {
>
> One way to solve this would be to call this remoteVideoResetTimer, and
> always cancel it when setting remoteVideo to a stream.
>
> I think we can get rid of the code block at line 243 with this change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/webrtc/pull/420/files#r23481519>.
>



-- 
Jiayang Liu